src/app/state/transaction/saga.ts

```diff
@@ -201,6 +201,40 @@ export function* doTransaction(action: PayloadAction<TransactionPayload>) {
 
     // Notify that the transaction was a success
     yield* put(transactionActions.transactionSent(action.payload))
+
+    const hash = yield* call([tw, tw.hash])
+
+    const transaction: Transaction = {
+      status: undefined,
+      fee: undefined,
+      level: undefined,
+      to: undefined,
+      round: undefined,
+      runtimeId: undefined,
+      runtimeName: undefined,
+      timestamp: undefined,
```

Comment:
I would save a timestamp

Reply:
You mean `new Date()`? If I understood correctly timestamp is a time when transaction was mined. Which would differ from the pending tx preview, so I would rather skip it.